themes/theme-default/src/node/defaultTheme.ts

```diff
@@ -62,6 +62,10 @@ export const defaultTheme = ({
             path.resolve(__dirname, '../client/components', file),
           ]),
       ),
+      '@theme/composables': path.resolve(
```

Comment:
I don't think this is a good idea. I assume your intention is to support overriding source files just like `@theme/*.vue`. However, the files pointed by `@theme/composables` export local files within the theme. Once `@theme/composables` is overwritten, it means all exports will be overridden, which may not meet your expectations and could lead to more issues.